### DISCUSSION TITLE
How can I run the C # project in this folder ：src.csharp

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1231] [Q&A] [answered]
I run  but show erro like this img

![未命名-3](https://github.com/CoderLine/alphaTab/assets/17884664/80d1b39a-4e9d-4134-b021-bc0aafc89269)

**Comment by Danielku15** [comment, marked as answer]
You might want to read through: https://www.alphatab.net/docs/contributing
If you wanna develop on alphatab and build the code yourself this is the place to get started. In your case you need to first translate the typescript code into C#.

**Comment by goodgame365** [comment]
thank  very much  I can run C# item ^_^
